tools/dev/gen_css_for_mne.py

```diff
@@ -16,11 +16,12 @@
 # Copyright the MNE-Python contributors.
```

Comment:
Snuck in a renaming of this file -- no reason to have it in `MNE` itself as it's really a packaging tool. This helps with stuff like automated dependency resolution as well (inspired by the autotick bot's analysis on conda-forge)